core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -721,11 +723,7 @@ protected function doReplace($strBuffer, $blnCache)
 
 				// Page
 				case 'page':
-					if (!$objPage->pageTitle && $elements[1] == 'pageTitle')
```

Comment:
shouldn't we continue to access the page model if there is no context? Also, how does this work BC if someone modifies the page model?

Reply:
It does work because the `PageModel` sets it on the context. And there cannot be no context because all legacy page types get one.